frontend/components/forms/fields/Dropdown/_styles.scss

```diff
@@ -251,10 +251,15 @@
       flex-grow: 1;
 
       .Select-value {
-        margin-top: 1px;
+        border: 1px solid $ui-fleet-black-75;
+        vertical-align: middle;
+        margin-top: 0;
+        display: inline-flex;
+        flex-direction: row-reverse;
```

Comment:
👍 this looks like a pretty detailed selector that shouldn't get picked up outside of the multi select.

Reply:
@lukeheath yeah, It's being used in select targets, schedule platform picker, packs platform picker, and add team member modal.

Reply:
@lukeheath Let me know if there's anything else that needs to be done!